test/parallel/test-compile-cache-relative-path.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+// This tests NODE_COMPILE_CACHE works with the NODE_COMPILE_CACHE_RELATIVE_PATH
+// environment variable.
+
+require('../common');
```

Comment:
Can you add some tests for `module.getCompileCacheDir()` as well? I think it should return a relative path (from `process.cwd()`?) in that case.